pkg/api/componentreadiness/component_report_test.go

```diff
@@ -1481,7 +1495,7 @@ func Test_componentReportGenerator_assessComponentStatus(t *testing.T) {
 			baseFlake:              1,
 			numberOfIgnoredSamples: 10,
 			expectedStatus:         -3,
-			expectedFischers:       1,
+			expectedFischers:       thrift.Float64Ptr(1),
```

Comment:
Nice find, I usually end up writing my own helpers for this kind of thing. Thrift looks pretty lightweight too